### DISCUSSION TITLE
posts/python-chicken-and-rabbit/

### DISCUSSION THREAD

**Discussion by giscus** [liwanggui/liwanggui.github.io#104] [Announcements]
# posts/python-chicken-and-rabbit/

使用 “穷举法” 求解鸡兔同笼，Python 代码实现如下 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 def chicken_and_rabbit(head: int, foot: int) -> list: if

https://liwanggui.com/posts/python-chicken-and-rabbit/

**Comment by wgleee** [comment]
*测试评论*
